src/common/contexts/helpers.tsx

```diff
@@ -1,6 +1,12 @@
 import { Magic } from "magic-sdk";
 import { MAGIC_API_KEY } from "../../config";
 
-export const magic = new Magic(MAGIC_API_KEY, {
-  network: "ropsten", // fix to ropsten network only
-});
+let magic: Magic | any;
```

Comment:
this is equal to `let magic: any`

What are you trying to do ? Should it be undefined ?

Reply:
thats true, but doing this allows for the autocomplete of methods i think. it should be undefined on the server side